lib/pymedphys/docs/lib/howto/gamma/from-dicom.ipynb

```diff
@@ -16,7 +16,8 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    class OOMFormatter(matplotlib.ticker.ScalarFormatter):
</pre><p>Is there a way to do this more simply? I'm pretty keen for these tutorials to be as easy to understand as achievable. At the end of the day, if fixing the ticker is going to be this complicated I'd prefer to leave the ticker in an untidy state for a tutorial such as this...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1591/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
To see the review comment inline within the notebook click on the "ReviewNB" link:

![image](https://user-images.githubusercontent.com/6559099/145162491-1fdf9d09-7e54-4de4-934e-aef07594a768.png)

Reply:
I agree with you. I also thought this is too complicated. I found this class online.
Anyway, I will find a simpler way.

Reply:
Yeah, ideally the tutorial would contain the minimal amount of code that achieves the desired outcome. Recognising that every piece of code in the tutorial someone new is going to be trying to understand what it is doing.

Reply:
Will update you tomorrow. :-)
Thank you so much!

Reply:
My pleasure :) Welcome to the land of git, GitHub, and PyMedPhys :) :tada:

Reply:
My pleasure :-)
I just updated again with a simpler way to show scientific notation, and updated the figure title for gamma index maps.